src/Symfony/Component/EventDispatcher/StandardEventListenerCaller.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\EventDispatcher;
+
+/**
+ *
+ */
+final class StandardEventListenerCaller implements EventListenerCallerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function call(callable $listener, Event $event, $eventName, EventDispatcherInterface $eventDispatcher)
+    {
+        $listener($event, $eventName, $eventDispatcher);
```

Comment:
``array(__CLASS__, 'someStaticMethod')`` cannot be called with this notation (this is the last remaining one in PHP 5.4+)